backend/globaleaks/rest/requests.py

```diff
@@ -15,8 +15,8 @@
 user_roles_regexp                 = r'^(admin|custodian|receiver)$'
 user_states_regexp                = r'^(enabled|disabled)$'
 receiver_img_regexp               = r'^([a-f0-9]{8}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{12}).png$'
-email_regexp                      = r'^([\w-]+\.)*[\w-]+@([\w-]+\.)+[a-z]{2,9}$|^$'
-email_regexp_or_empty             = r'^([\w-]+\.)*[\w-]+@([\w-]+\.)+[a-z]{2,9}$|^$'
+email_regexp                      = r'^(([\w-]\.){0,100}[\w-]{1,100}@([\w-]+\.){0,100}[\w-]{1,100})$|^$'
```

Comment:
thank you for noticing it. i'm going to take a look to this issue while integrating.